src/remediations/fifi.js

```diff
@@ -526,7 +686,9 @@ exports.createPlaybookRun = async function (status, remediation, username, exclu
         const rhcRequests = splitRequests.RHC[0]; // There should only ever be one
         await dispatchRHCRequests(rhcRequests, playbook_run_id);
 
-        // TODO: Store responses??
+        if (_.isEmpty(splitRequests.satellite)) {
```

Comment:
why the isEmpty() check?

Reply:
this is for the situation such that there are no satellite runs defined.  When that is the case we store the RHC only run as just the "playbook_run" in the remediations db.